cunumeric/random/random.py

```diff
@@ -43,6 +50,50 @@ def beta(
     size: Union[NdShapeLike, None] = None,
     dtype: npt.DTypeLike = np.float64,
 ) -> ndarray:
+    """
+    beta(a, b, size=None)
+
+    Draw samples from a Beta distribution.
+
+    The Beta distribution is a special case of the Dirichlet distribution,
+    and is related to the Gamma distribution.  It has the probability
+    distribution function
+
+    .. math:: f(x; a,b) = \\frac{1}{B(\\alpha, \\beta)} x^{\\alpha - 1}
+                                                     (1 - x)^{\\beta - 1},
+
+    where the normalization, B, is the beta function,
+
+    .. math:: B(\\alpha, \\beta) = \\int_0^1 t^{\\alpha - 1}
+                                 (1 - t)^{\\beta - 1} dt.
+
+    It is often seen in Bayesian inference and order statistics.
+
+    Parameters
+    ----------
+    a : float or array_like of floats
+        Alpha, positive (>0).
+    b : float or array_like of floats
+        Beta, positive (>0).
+    size : int or tuple of ints, optional
+        Output shape.  If the given shape is, e.g., ``(m, n, k)``, then
+        ``m * n * k`` samples are drawn.  If size is ``None`` (default),
+        a single value is returned if ``a`` and ``b`` are both scalars.
+        Otherwise, ``np.broadcast(a, b).size`` samples are drawn.
```

Comment:
If `a` and `b` can only be scalars, then this should be changed to
```
If size is ``None`` (default), a single value is returned.
```
Similarly for other functions in this file with this behavior.

Reply:
3f68ce4aacfbb28323d4ef611e7badaccfed9ac4